### DISCUSSION TITLE
PLEASE DO NOT VISIT PORN WEBSITES!!!!!

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#6] [Announcements]
I just watched someone "accidentally" open a furry porn site on Clockwork, and I just wanted to let you all know not to visit these websites using Clockwork. You'll get us blocked. Please don't get us blocked.

**Comment by ghost** [comment]
Ok thnx

**Comment by skylareight** [comment]
how do i use it? it only shows the screen where its like a bunch of computers and quotes and i cant figure out how to get past that.
